.gitattributes

```diff
@@ -0,0 +1,15 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
```

Comment:
I'm not sure how much we will be using visual studio but it might be useful to add these other file extensions https://github.com/alexkaratarakis/gitattributes/blob/master/VisualStudio.gitattributes

That repo has a bunch of ready made .gitattributes files